xls/modules/zstd/external/BUILD

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 The XLS Authors
```

Comment:
can. you add a docstring about what those external are and why they are needed?

Reply:
Added the docstring and a License file for the third party sources